compiler/src/test/java/tiamat/compiler/PrefsProcessorTest.java

```diff
@@ -18,12 +18,13 @@
 
     @Test
     public void tableNameNotDefined() {
```

Comment:
One idea comes to mind is to create a new android library module(say, "compiler-test") and move test cases to that project. 
Since new module is android library project, we should be able to import both "compiler" and "library" module and run tests without problem.

But I hope there are any other nicer solutions...